src/LICENSE

```diff
@@ -0,0 +1,30 @@
+Copyright Alessandro de Gennaro and Andrey Mokhov (c) 2017
```

Comment:
While we are doing the move, let's also change the license to simpler MIT license with the copyright to Tuura team. You can simply copy this file: https://github.com/tuura/centrifuge/blob/master/LICENSE